src/test/java/com/gooddata/http/client/GoodDataHttpClientIntegrationTest.java

```diff
@@ -92,7 +93,7 @@ public void getProjectsBadLogin() throws IOException {
                 .withContentType(CONTENT_TYPE_JSON);
 
         final HttpClient client = createGoodDataClient(jadlerLogin, jadlerPassword, jadlerHost);
-
+        assertNotNull(client);
```

Comment:
same here, if there will be some client related fields null, the exception would be fired sooner